hubblestack/extmods/module_runner/runner_factory.py

```diff
@@ -0,0 +1,18 @@
+from hubblestack.extmods.module_runner.audit_runner import AuditRunner
+from hubblestack.extmods.module_runner.fdg_runner import FdgRunner
+from hubblestack.extmods.module_runner.runner import Caller
+
+import logging
+log = logging.getLogger(__name__)
+
+def get_audit_runner(salt_dict, grains_dict, opts_dict):
```

Comment:
As salt_dict is __salt__, grains_dict is __grains__ and opts_dict is __opts__ which are global in context of hubble.
So we don't require them to be passed as params as they are already available in all modules.

Reply:
They were not available there. I fixed it now. Needed to define them in daemon